packages/application-config/custom-view.schema.json

```diff
@@ -0,0 +1,206 @@
+{
```

Comment:
Custom Views have a slightly different configuration requirements than Custom Apps, so that's why we renamed the previous schema and created this one.